### PR TITLE
Update ruleset

### PR DESCRIPTION
**Pull request by kalpetros**
Set Greek-travel ruleset to disabled
by default due to problems it is causing

### CODE REVIEW

File: src/chrome/content/rules/Greek-travel.xml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 <!-- This ruleset is experimental. If you experience problems please
-	 open an issue at https://github.com/kargig/https-everywhere-greek-rules -->
+open an issue at https://github.com/kargig/https-everywhere-greek-rules. -->
+<!-- Update 01/March/2016: Greek-travel set to disabled by default due to problems it is causing -->
 
-<ruleset name="Greek-travel">
-  <target host="*.aegeanair.com" />
+<ruleset name="Greek-travel (buggy)" default_off="breaks site">
```
Comment (J0WI) [inline src/chrome/content/rules/Greek-travel.xml]: seems to be `http redircet`

**Comment by J0WI** [comment]
@kalpetros could you update the cause?
I'll merge it then.

**Comment by kalpetros** [comment]
@J0WI I updated my pull request. It seems that this ruleset triggers mixed content and breaks a big part of the site.

**Comment by J0WI** [comment]
Please replace wildcard and regex or provide some test urls to pass our tests:
https://github.com/EFForg/https-everywhere/blob/master/ruleset-testing.md

**Comment by bardiharborow** [comment]
@J0WI Fixed by #4931.

**Pull request closed without merging**